### PR TITLE
add crud operations for contacts table

### PR DESCRIPTION
**Pull request by SUSHILkhadka**

### CODE REVIEW

File: src/db/migrations/20220811170457_create_refresh_token_table.ts [hunk 1]
```diff
@@ -1,17 +1,13 @@
...
 
 export async function up(knex: Knex): Promise<void> {
-    return knex.schema.createTable('refresh_token', (table) => {
-        table.string('refreshToken').notNullable();
-        table.string('id').notNullable();
-        table.string('expiresAt').notNullable();
-      });
+  return knex.schema.createTable('refresh_token', (table) => {
+    table.string('refreshToken').notNullable();
+    table.string('id').notNullable();
```
Comment (aman007shrestha) [inline src/db/migrations/20220811170457_create_refresh_token_table.ts:6]: Numerical Id would be better in my opinion. They are lot more fast in terms of query speed compared to string.

File: src/models/userModel.ts [hunk 2]
```diff
@@ -25,5 +25,18 @@ class User {
     const users = await db(this.table).select().returning('*');
```
Comment (aman007shrestha) [inline src/models/userModel.ts:25]: I think this part will return all the users along with their hashed password? Is that is the case, we would not want any user to view others' password in their response. IMPORTANT that u address it.

**Review by aman007shrestha — APPROVED** [review]
Review the comments. Except that Looks good to me.

File: src/models/userModel.ts (hunk 2 above)
Comment (SUSHILkhadka) [reply src/models/userModel.ts:25]: One user will not get other user's passwordHash as response, as I would delete getAllUsers route later (it is easy to use route without any body for authentication check, so using it for testing). But still I will try to not return passwordHash as response whenever possible. Thanks for advice.

File: src/db/migrations/20220811170457_create_refresh_token_table.ts (hunk 1 above)
Comment (SUSHILkhadka) [reply src/db/migrations/20220811170457_create_refresh_token_table.ts:6]: ok.

**Review by Panas18 — APPROVED** [review]
LGTM

**Review by crazieeKinG — APPROVED** [review]

**Pull request merged**